### PR TITLE
Feature/easy peasy

### PR DESCRIPTION
**Pull request by RenzoLetona**
# Title

- Please include the Story name

## Related Story

- Link to the Story

## Info / Context

- Description of feature, bug, or desired alteration

## Changes

- Change 1
- Change 2

## Testing

- [x] Includes test coverage

## Manual testing steps

- Procedure for reviewers

### CODE REVIEW

File: models/model.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { action, thunk } from 'easy-peasy'
```
Comment (facucaramelpoint) [inline models/model.tsx:1]: the file name could reference to `food`

File: components/ShopList/Foods.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+`
+
+const Todos = () => {
+  const foods = useStoreState((state: FoodModel) => state.foods)
+  const update = useStoreActions((actions: Actions<FoodModel>) => actions.updateThunk)
+  return (
+    <InfoBox>
+      <h1>Food shopping´s list</h1>
+      <p>Click an item to cross it out</p>
+      <TextBtn onClick={() => update()}>Activate thunk</TextBtn>
```
Comment (facucaramelpoint) [inline components/ShopList/Foods.tsx:42]: `onClick={update}` should work equals, and avoid creating a function on every re-render

**Review by facucaramelpoint — APPROVED** [review]

File: components/ShopList/Foods.tsx [hunk 3]
```diff
@@ -39,7 +39,7 @@ const Todos = () => {
     <InfoBox>
       <h1>Food shopping´s list</h1>
       <p>Click an item to cross it out</p>
-      <TextBtn onClick={() => update()}>Activate thunk</TextBtn>
+      <TextBtn onClick={update()}>Activate thunk</TextBtn>
```
Comment (facucaramelpoint) [inline components/ShopList/Foods.tsx:42]: `update` without invoking the function

**Review by facucaramelpoint — CHANGES_REQUESTED** [review]

File: components/ShopList/Foods.tsx (hunk 3 above)
Comment (RenzoLetona) [reply components/ShopList/Foods.tsx:42]: that doesn´t recognize the function without ()
Captura de Pantalla 2022-07-05 a la(s) 16 43 47

**Pull request merged**